Problem: (cro-1231) no tx-building in ios example using c-bindings

leejw51crypto opened the pull request:
Solution: add low level tx-building api (unbond, withdraw, transfer, deposit)

tomtau reviewed (changes requested):
there seem to be some conflicts with mater?

leejw51crypto commented:
i'll check

leejw51crypto committed:
Problem: (cro-1231) no tx-building in ios example using c-bindings

Solution: add low level tx-building api (unbond, withdraw, transfer, deposit)
clear data


tidy up


tidy up
